src/components/chat/components/sendOfferPopup.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { FunctionComponent, useState } from 'react';
+import ReactDOM from 'react-dom';
+import { yupResolver } from '@hookform/resolvers/yup';
+import {
+	Wrapper,
+	StyledModal,
+	CloseButton,
+	Content,
+	Backdrop,
+	Form,
+	Label,
+	P,
+} from 'components/PostDetailsPage/components/Modal.styles';
+import { ButtonBlock, ContainerDate, DateBlock, Title } from './sendOffer.styles';
+import { Input } from 'components/clientSettings/clentSettings.styles';
+import { Cancel, Button } from 'pages/setting-page-client/change-password/ChangePassword.styles';
+import { t } from 'i18next';
+import { useGetClientInfoByUserQuery, usePostOfferMutation } from 'service/httpService';
+import { notification } from 'antd';
+import { useAppSelector } from 'redux/hooks';
+import { RootState } from 'redux/store';
+import { useForm } from 'react-hook-form';
+import { ModalProps, NotificationType, OfferForm, Schema } from './data';
+
+export const SendOfferPopup: FunctionComponent<ModalProps> = ({
+	isShown,
+	setIsShown,
+	hide,
+	freelancerId,
+	jopPostId,
+}) => {
+	const {
+		register,
+		handleSubmit,
+		reset,
+		formState: { errors },
+	} = useForm<OfferForm>({
+		resolver: yupResolver(Schema),
+	});
+	const [sendForm] = usePostOfferMutation();
+	const { user } = useAppSelector<RootState>(state => state);
+	const { data: clientInfo } = useGetClientInfoByUserQuery(user.id);
+
+	const resetInput = () => {
+		return reset({ price: 0, startDate: {}, endDate: {} });
+	};
+
+	const openNotificationWithIcon = (type: NotificationType) => {
+		notification[type]({
+			message:
+				type === 'success' ? `${t('SendOfferPopup.success')}` : `${t('SendOfferPopup.error')}`,
```

Comment:
1) cont compare with string, 2) never compare with translations file. only constants could be compared

Reply:
fixed